### ISSUE TITLE
3.5.9.9 fails to start when gridcoinresearch.conf is not present

### ISSUE THREAD

**Issue by Bouowmx** [gridcoin-community/Gridcoin-Research#488]
Windows 10 1703 64-bit:

How to reproduce:
1. Go to `%appdata%\GridcoinResearch`
2. Rename `gridcoinresearch.conf` to something else
3. Start `gridcoinresearch.exe`, and it will quit very quickly after

Contents of `debug.log`: My user-specific path has been replaced with `%appdata%`
```


Starting Gridcoin

Initializing GUI...

Boost Version: Using Boost 1.55.0

Using specific arg default



















08/06/17 19:53:45 ***************************************** GRIDCOIN RESEARCH ***************************************************

08/06/17 19:53:45 Gridcoin version v3.5.9.9-g-research (12/1/2014)
08/06/17 19:53:45 Using OpenSSL version OpenSSL 1.0.1p 9 Jul 2015
08/06/17 19:53:45 Default data directory %appdata%\GridcoinResearch
08/06/17 19:53:45 Used data directory %appdata%\GridcoinResearch
08/06/17 19:53:45 dbenv.open LogDir=%appdata%\GridcoinResearch\database ErrorFile=%appdata%\GridcoinResearch\db.log
08/06/17 19:53:45 Loading block index...
08/06/17 19:53:45 Mode=Prod

08/06/17 19:53:45 Opening LevelDB in %appdata%\GridcoinResearch\txleveldb
08/06/17 19:53:45 Transaction index version is 180015
08/06/17 19:53:45 Opened LevelDB successfully
08/06/17 19:53:45 Loading DiskIndex 0

```

**Comment by tomasbrod** [comment]
Just like that? No errors?
Also can you try with empty data directory (make backup first)?

**Comment by chadunit** [comment]
Hi, I'm new to Gridcoin/BOINC as of yesterday. As a first-time install from scratch, I'm also experiencing this on 2 Windows 10 systems. The gridcoinresearch.exe process starts, a brief splash screen shows, and then it dies within about 1 or 2 seconds. No errors displayed, nothing in the Windows Event Logs.

I don't have any gridcoinresearch.conf file in the data dir or the program installation dir. I don't know where that's supposed to come from or if/when it's supposed to be automatically created. After emptying the data dir, the next launch of the process recreates multiple files but no gridcoinresearch.conf, and still has the same failure.

My debug.log is as follows:

```
Starting Gridcoin

Initializing GUI...

Boost Version: Using Boost 1.55.0

Using specific arg default



















08/06/17 21:48:19 ***************************************** GRIDCOIN RESEARCH ***************************************************

08/06/17 21:48:19 Gridcoin version v3.5.9.9-g-research (12/1/2014)
08/06/17 21:48:19 Using OpenSSL version OpenSSL 1.0.1p 9 Jul 2015
08/06/17 21:48:19 Default data directory C:\Users\MYUSERNAME\AppData\Roaming\GridcoinResearch
08/06/17 21:48:19 Used data directory C:\Users\MYUSERNAME\AppData\Roaming\GridcoinResearch
08/06/17 21:48:19 dbenv.open LogDir=C:\Users\MYUSERNAME\AppData\Roaming\GridcoinResearch\database ErrorFile=C:\Users\MYUSERNAME\AppData\Roaming\GridcoinResearch\db.log
08/06/17 21:48:20 Loading block index...
08/06/17 21:48:20 Mode=Prod

08/06/17 21:48:20 Opening LevelDB in C:\Users\MYUSERNAME\AppData\Roaming\GridcoinResearch\txleveldb
08/06/17 21:48:20 Transaction index version is 180015
08/06/17 21:48:20 Opened LevelDB successfully
08/06/17 21:48:20 Loading DiskIndex 0
08/06/17 21:48:20 Time to memorize diskindex containing 1 blocks :               0ms

08/06/17 21:48:20 Time to calculate Chain Trust               0ms
08/06/17 21:48:20 LoadBlockIndex(): hashBestChain=000005a247b397eadfef  height=0  trust=1048577  date=10/11/14 13:22:57
08/06/17 21:48:20 LoadBlockIndex(): synchronized checkpoint 000005a247b397eadfefa58e872bc967c2614797bdc8d4d0e6b09fea5c191599
08/06/17 21:48:20 Verifying last 0 blocks at level 1
08/06/17 21:48:20 Time to Verify Blocks               0ms
08/06/17 21:48:20 Set up RA RA Complete - RA Time               0ms
08/06/17 21:48:20  block index              22ms
08/06/17 21:48:20 Loading wallet...
```

**Comment by iFoggz** [comment]
yes. when @denravonska  on we will discuss this along with what i may of found.

**Comment by iFoggz** [comment]
I have found the bug a PR is up awaiting approval with solution. #491

**Comment by denravonska** [comment]
Fixed with #491.
